manifests/kots-config.yaml

```diff
@@ -432,6 +432,15 @@ spec:
             If you don't provide an Allocation ID, a random domain will be assigned to your load balancer. You can find some information about how to use the domain you configured for CARTO platform in the [AWS ELB documentation](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/using-domain-names-with-elb.html).
           recommended: true
 
+        # Ingress testing mode
+        - name: enableIngressTestingMode
+          title: Ingress testing mode
+          when: '{{repl or (ConfigOptionEquals "accessToCartoModeK8s" "accessToCartoModeK8sCustom") (ConfigOptionEquals "accessToCartoModeK8sNotFullySupported" "accessToCartoModeK8sCustom") (ConfigOptionEquals "isEmbeddedCluster" "true") }}'
+          help_text: |-
+            Enable if you want to enable ingress testing mode. This will just deploy the CARTO router in testing mode to check that the layers you place on top of CARTO work.
```

Comment:
Exactly 😄 It will show a dummy website. The changes are added in the router-http image here: https://github.com/CartoDB/cloud-native/pull/16898 Basically we change the Nginx config to route all requests to a static website.

The layers that can be placed on top of CARTO are on the customer side. We offer the default ingress method were you don't have to worry about anything and the custom method.

In the custom method, customer can terminate the TLS at the router level providing their certificate and key, or they can serve traffic using HTTP and terminate the TLS at a higher layer.

In that particular scenario is where this ingress testing mode helps, as we'll be able to ask customer to do a port-forward to the router service and if the static website appear, then they have to check whatever they're putting on top of the SH if it doesn't work :)

Reply:
A suggestion, or something similar
```suggestion
            Enable if you want to turn on ingress testing mode. This will deploy the CARTO router in testing mode, which shows a hello-world page, without redirecting traffic to the internal components, to check that the layers you place on top of CARTO work.
```